### ISSUE TITLE
Non-empty ldconfig post claims there is no ldconfig

### ISSUE THREAD

**Issue by jengelh** [rpm-software-management/rpmlint#550]
```
%post -n libblah0 -p /sbin/ldconfig
echo hi world
```

This correctly elicits the `libblah0.x86_64: E: non-empty-%post /sbin/ldconfig` warning, but it also causes a false positive:

```
[   23s] libblah0.x86_64: E: postin-without-ldconfig /usr/lib64/libblah.so.0.0.0
[   23s] This package contains a library and its %post scriptlet doesn't call ldconfig.
```

The package certainly calls ldconfig - just not the right way :)

**Comment by scarabeusiv** [comment]
This should really be moved to file triggers.

**Comment by Conan-Kudo** [comment]
Assuming a [transition from Zypper to DNF occurs in openSUSE](https://lists.opensuse.org/archives/list/factory@lists.opensuse.org/thread/GAPGVVKWI5CCCKMR6JQI5RMEOZNTOMJK/), we should be able to finally move to file triggers like Fedora and Mageia did long ago...
